packages/cmf/.npmignore

```diff
@@ -37,3 +37,23 @@ jspm_packages
 .node_repl_history
 
 __tests__
+
+storybook-static
+*.tgz
+src
+screenshots
+yarn.lock
+stories
+stories-core
+e2e
+screenshots.config.json
+.sass-lint.yml
+.storybook
+output
+.eslintrc
+examples
+.yarn-metadata.json
+__mocks__
+.editorconfig
+.babelrc
+test
```

Comment:
this file is not really representative of root file hierarchy :/
Should it useful for forms packages?